autocompletefilter/filters.py

```diff
@@ -73,7 +73,12 @@ def choices(self, changelist):
 
         lookup_display = None
         if self.lookup_val:
-            instance = self.field.related_model.objects.get(pk=self.lookup_val)
+            if django.VERSION >= (5, 0):
```

Comment:
I would prefer to have code that doesn't rely on version checks. Is it possible we could replace this with a type check `isinstance(self.lookup_val, list)`?

Reply:
Makes sense - I've made this change.

Thanks, I appreciate the effort you folks make in making and maintaining this package :)

Is there anything I can do to expedite getting a new version of this released?

Reply:
> I would prefer to have code that doesn't rely on version checks. Is it possible we could replace this with a type check `isinstance(self.lookup_val, list)`?

Can I change your mind?
https://adamj.eu/tech/2020/01/07/feature-checking-versus-version-checking/

You'll get cleaner code with version checking, and once you remove support for the old version, you can easily remove it. And here you hide it implicitly. Which is somewhat contrary to the zen of Python

> Explicit is better than implicit.

Reply:
@Mogost thanks, it does sound reasonable. The previous iteration had both the version and type check. Would you still include the latter as well? Because with the check it looks like the application can come across both variants, no matter the Django version.

Reply:
I do not think it is possible to face the list in Django lower 5.0. Also I do not see a possibility to face not a list in 5.0+.

@niallcm could you please change the code one more time? Sorry for the noise

Reply:
Shall I bump the version number while I'm at it?